data-import.qmd

```diff
@@ -312,16 +331,19 @@ csv <- "
 If we read it without any additional arguments, `x` becomes a character column:
 
 ```{r}
-df <- read_csv(csv)
+df <- read_csv(simple_csv)
 ```
 
 In this very small case, you can easily see the missing value `.`.
 But what happens if you have thousands of rows with only a few missing values represented by `.`s speckled among them?
 One approach is to tell readr that `x` is a numeric column, and then see where it fails.
-You can do that with the `col_types` argument, which takes a named list:
+You can do that with the `col_types` argument, which takes a named list where the names match the column names in the CSV file:
 
 ```{r}
-df <- read_csv(csv, col_types = list(x = col_double()))
+df <- read_csv(
+  simple_csv, 
+  col_types = list(x = col_double())
+  )
```

Comment:
```suggestion
)
```

BTW have you tried http://github.com/lionel-/codegrip ?

Reply:
Yes, though haven't gotten to a place where I use it regularly, clearly need to!